src/modules/common/components/FailureSwitcher.js

```diff
@@ -32,6 +34,15 @@ export class FailureSwitcher extends React.Component<PropsType> {
         return <Failure goToPath={new EventsRouteConfig().getRoutePath({ city, language })}
                         goToMessage='goTo.events'
                         errorMessage='notFound.event' />
+      case 'newsItem':
```

Comment:
I suggest to move switch cases onto constants

Reply:
The existed code using texts so I think we should stick to this. And since this is used in one place only I don't see an issue in having it as text

Reply:
That's fine